README.md

```diff
@@ -49,6 +49,13 @@ which generally keeps tests cleaner / clearer.
 
 ## API
 
+Weaver provides two types of testing suites.
```

Comment:
Slightly inaccurate : there are more than two (in particular if you include zio). Not sure how to phrase it whilst not getting into details. Maybe a "mainly" dropped in there would suffice

Reply:
I found the `zio` stuff and added a new section to clarify (and clarified the original wording).

Reply:
Perfect, thanks !